escrow/escrow.go

```diff
@@ -216,17 +216,19 @@ func (payment *paymentTransaction) Commit() error {
 	e := payment.service.storage.Put(
 		&PaymentChannelKey{ID: payment.payment.ChannelID},
 		&PaymentChannelData{
-			ChannelID:        payment.channel.ChannelID,
-			Nonce:            payment.channel.Nonce,
-			State:            payment.channel.State,
-			Sender:           payment.channel.Sender,
-			Recipient:        payment.channel.Recipient,
-			FullAmount:       payment.channel.FullAmount,
-			Expiration:       payment.channel.Expiration,
-			Signer:           payment.channel.Signer,
-			AuthorizedAmount: payment.payment.Amount,
-			Signature:        payment.payment.Signature,
-			GroupID:          payment.channel.GroupID,
+			ChannelID:            payment.channel.ChannelID,
+			Nonce:                payment.channel.Nonce,
+			State:                payment.channel.State,
+			Sender:               payment.channel.Sender,
+			Recipient:            payment.channel.Recipient,
+			FullAmount:           payment.channel.FullAmount,
+			Expiration:           payment.channel.Expiration,
+			Signer:               payment.channel.Signer,
+			AuthorizedAmount:     payment.payment.Amount,
+			Signature:            payment.payment.Signature,
+			GroupID:              payment.channel.GroupID,
+			OldnonceSignedAmount: payment.channel.OldnonceSignedAmount,
```

Comment:
Minor, but I would use `OldNonceSignetAmount` and `OldNonceSignature` as it looks a bit better.